File: combine-handler/src/main/java/cc/vileda/kalfor/handler/KalforProxyHeader.java
package cc.vileda.kalfor.handler;

public class KalforProxyHeader
{
	@SuppressWarnings("WeakerAccess")
	public String name;

	@SuppressWarnings("WeakerAccess")
	public String value;

	@SuppressWarnings("unused")
	public KalforProxyHeader()
	{
	}

	public KalforProxyHeader(final String name, final String value)
	{
		this.name = name;
		this.value = value;
	}
}

File: library/src/main/kotlin/cc/vileda/kalfor/handler/KalforProxyHeader.kt
package cc.vileda.kalfor.handler

@Suppress("unused")
class KalforProxyHeader {
    var name: String = ""

    var value: String = ""

    constructor() {
    }

    constructor(name: String, value: String) {
        this.name = name
        this.value = value
    }
}
